client/src/components/Form/components/formSections/DiagnoseSection/DiagnoseSection.tsx

```diff
@@ -68,6 +68,8 @@ const DiagnoseSection = ({ section, setSchema, schema, errors, validate, diagnos
                                 const updatedSchema = {
                                     ...state,
                                     annenFraversArsak: !state.annenFraversArsak,
+                                    annenFraversArsakGrunn: undefined,
+                                    annenFraversArsakBeskrivelse: undefined,
                                 };
                                 validate('annenFraversArsak', updatedSchema);
                                 return updatedSchema;
```

Comment:
```suggestion
                                validate('annenFraversArsak', updatedSchema);
                                validate('annenFraversArsakGrunn', updatedSchema);
                                validate('annenFraversArsakBeskrivelse', updatedSchema);
                                return updatedSchema;
```
Vi har en liten issue her, hvor man kan få feil på underspørsmål, som ikke går vekk før man prøver å submitte. Den foreslåtte endringen her er heller ikke så nice, da det vil komme feil på underspørsmål med én gang... Vi burde kanskje hatt en state i formen `attemptedSubmittet: boolean`. Hva tror du?

Reply:
for å gjenskape feil:

1. klikk vekk alle valg under 3.3.1 Lovfestet fraværsgrunn
2. klikk vekk 3.1